packages/dds/tree/src/simple-tree/core/withType.ts

```diff
@@ -39,6 +42,27 @@ export const typeNameSymbol: unique symbol = Symbol("TreeNode Type");
  */
 export const typeSchemaSymbol: unique symbol = Symbol("TreeNode Schema");
 
+/**
+ * The intended type of insertable content that is to become a {@link TreeNode}.
+ * @remarks **Note:** Whenever possible, use the type-safe {@link (TreeAlpha:interface).ensureSchema} function rather than manually employing this symbol.
```

Comment:
Do we need to export this at all? Or can we leave it an implementation detail of `ensureSchema` (at least for now)?

Reply:
Good question. My thought was that if somebody were to run into one of those "Ahh the compile time checking of `ensureSchema` isn't working because of "whatever reasons" (recursive types or something as crazy), then they have the fallback of using this symbol. But maybe the fallback in that case should just be to do a cast. What do you think? It's easy enough to slap this symbol on an object, but it's hard to put it on an array or a map.

Reply:
I guess my opinion is generally not to expose something until we identify a need. But it's just alpha, so I wouldn't block either way. Your call.